cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss

```diff
@@ -409,48 +402,11 @@ a {
     height: $loading-width;
 }
 
-// Search page
-
-.badges-box {
-    font-size: .9em;
-
-    .ui.label {
-        &.badge {
-            padding: .4em .7em;
-            font-size: .7em;
-            font-weight: $font-heavy;
-            position:relative;
-        }
-
-        &.badge:hover {
-            text-decoration: none;
-            color: white;
-        }
-    }
-}
+// Labels
 
 .ui.label {
-    &.badge-type {
-        background-color: $light-blue;
-        color: white;
-
-        &:hover {
-            background-color: darken($light-blue, $darken-on-hover);
-            color: white;
-        }
-
-    }
-    &.badge-subtype {
-        background-color: $medium-blue;
-        color: white;
-
-        &:hover {
-            background-color: darken($medium-blue, $darken-on-hover);
-            color: white;
-        }
-    }
 
-    &.badge-category {
+    &.dark-cyan {
         background-color: #0f6d7c;
```

Comment:
Minor: what do you think about moving these colors to [`variables.scss`](https://github.com/cernopendata/opendata.cern.ch/blob/master/cernopendata/modules/theme/assets/semantic-ui/scss/variables.scss) for the sake of reusability?